Core/Source/MVVM/ViewModelUserEvents.swift

```diff
@@ -19,6 +19,9 @@ public enum ViewModelUserEvent {
 
     /// On touch platforms, represents the target receiving a single tap.
     case tap
+
+    /// Represents the user attempting to copy the view model to the pasteboard.
+    case copy
```

Comment:
does this need `EventKeyModifierFlags`?

Reply:
It's just tied to the system menu item with the `copy(_:)` action firing so we don't actually get any key event